lua/java/ui/profile.lua

```diff
@@ -144,9 +145,6 @@ function ProfileUI:get_menu()
 			close = { '<Esc>', '<C-c>' },
 			submit = { '<CR>', '<Space>' },
 		},
-		on_change = function(item)
-			self.focus_item = item
```

Comment:
On change callback is not being called on cursor change so the `focus_item` is the top most node when the window is opened. As a result deletion fails because `focus_item` is always the `active` item